setup.py

```diff
@@ -28,7 +28,7 @@
 
 
 install_requires = [
-    'urllib3==1.23',  # https://github.com/requests/requests/issues/4830
+    'urllib3>=1.23',
```

Comment:
You could add <2 if you want to assume that they are fallowing semver.

Reply:
Done.